librz/include/rz_analysis.h

```diff
@@ -600,11 +599,30 @@ typedef struct rz_analysis_bind_t {
 
 typedef const char *(*RzAnalysisLabelAt)(RzAnalysisFunction *fcn, ut64);
 
-typedef enum {
-	RZ_ANALYSIS_VAR_KIND_REG = 'r',
-	RZ_ANALYSIS_VAR_KIND_BPV = 'b',
-	RZ_ANALYSIS_VAR_KIND_SPV = 's'
-} RzAnalysisVarKind;
+/**
+ * \brief An address on the stack
+ *
+ * These addresses are a relative offset to the value of the stack pointer
+ * when entering a function, i.e. just before the first instruction inside the function is executed.
+ * Thus, with the most common calling conventions, the return address with be at addr 0,
+ * arguments at positive addresses and local variables at negative addresses:
+ *
```

Comment:
So it means that if I have something like:
```
push r15
push r14
push rbx
sub rsp, 0x50
mov qword [rsp + 10], 0x22
```

the variable at `rsp+10` will be marked as *-88*, right? 8(r15) + 8 (r14) + 8(rbx) + 0x50 - 0x10 == 88 == 0x58.

Did I get it right?

Reply:
Exactly.